__tests__/app.test.js

```diff
@@ -213,6 +213,54 @@ describe('PATCH commands', () => {
   
 });
 
+describe('Get /api/reviews Queries', () => {
+    test('200: should return the reviews with the category dexterity', () => {
```

Comment:
This needs to be rephrased as something like "should return the reviews of the category passed"